sql/mob_skill_lists.sql

```diff
@@ -3531,73 +3571,195 @@ INSERT INTO `mob_skill_lists` VALUES ('TRUST_Maat',1048,1028); -- Shoulder Tackl
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Karaha-Baruha',1051,180);      -- Sunburst
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Karaha-Baruha',1051,183);      -- Spirit Taker
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Karaha-Baruha',1051,3336);  -- Howling Moon
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Karaha-Baruha',1051,3337);  -- Lunar Bay
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Cid',1052,166);                -- True Strike
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Cid',1052,168);                -- Hexa Strike
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Cid',1052,3322);            -- Critical Mass
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Cid',1052,3323);            -- Fiery Tailings
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Gilgamesh',1053,146);          -- Tatsun: Goten
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Gilgamesh',1053,152);          -- Tachi: Kasha
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Gilgamesh',1053,3434);      -- Tachi: Kamai
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Gilgamesh',1053,3435);      -- Iaikuki
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Areuhat',1054,37);             -- Seraph Blade
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Areuhat',1054,40);             -- Vorpal Blade
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Areuhat',1054,42);             -- Savage Blade
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Areuhat',1054,3438);           -- Dragon Breath
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Areuhat',1054,3439);           -- Hurricane Wing
 INSERT INTO `mob_skill_lists` VALUES ('TRUST_Semih_Lafihna',1055,3487); -- Sidewinder (Semih Lafihna)
 INSERT INTO `mob_skill_lists` VALUES ('TRUST_Semih_Lafihna',1055,3488); -- Arching Arrow (Semih Lafihna)
 INSERT INTO `mob_skill_lists` VALUES ('TRUST_Semih_Lafihna',1055,3489); -- Lux Arrow
 INSERT INTO `mob_skill_lists` VALUES ('TRUST_Semih_Lafihna',1055,3490); -- Stellar Arrow
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Elivira',1056,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Noillurie',1057,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Lhu_Mhakaracca',1058,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Ferreous_Coffin',1059,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Lilisette',1060,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Mumor',1061,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Uka_Totlihn',1062,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Klara',1063,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Romaa_Mihgo',1064,0);
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Elivira',1056,209);            -- Split Shot
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Elivira',1056,212);            -- Slug Shot
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Elivira',1056,214);            -- Heavy Shot
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Elivira',1056,1201);           -- Coronach
```

Comment:
Wrong animation

Reply:
The entire table was prefilled with wrong IDs more than 6 years ago and everyone has been correcting them one at a time as they are implemented.

Really do not need a pr comment for every individual row for previously unimplemented skills that don't have a correct animation. just don't implement without getting the right one - that blows up everyone's inboxes so comment the row, not the PR.

Reply:
p.s. allowing wrong animation IDs to activate on a running server can crash clients, so I really do not wish to see skills enabled with wrong IDs.